src/View3d.ts

```diff
@@ -734,13 +735,13 @@ export class View3d {
    * Set clipping range (between 0 and 1) for a given axis.
    * Calling this allows the rendering to compensate for changes in thickness in orthographic views that affect how bright the volume is.
    * @param {Object} volume
-   * @param {number} axis 0, 1, or 2 for x, y, or z axis
+   * @param {number} axis x, y, or z axis
```

Comment:
looks like the type should be string here?

Reply:
This comment is only because the type in the actual function args is different, so maybe that is the one to change? something is out of sync here.